pkgs/servers/sql/postgresql/ext/tds_fdw.nix

```diff
@@ -0,0 +1,29 @@
+{ stdenv, fetchFromGitHub, postgresql, freetds }:
+
+stdenv.mkDerivation rec {
+  name = "tds_fdw-${version}";
```

Comment:
name will automatically be generated if you do:
```suggestion
  pname = "tds_fdw";
```

Reply:
Done!